packages/babel-helpers/src/helpers.js

```diff
@@ -449,7 +449,7 @@ helpers.construct = () => template.program.ast`
     if (Reflect.construct.sham) return false;
 
     // Proxy can't be polyfilled. Every browser implemented
-    // proxies before or at the same time of Reflect.construct,
+    // proxies before or at the same time as Reflect.construct,
     // so if they support Proxy they also support Reflect.construct.
     if (typeof Proxy === "function") return true;
```

Comment:
> Every browser implemented proxies before or at the same time as `Reflect.construct`, so if they support `Proxy` they also support `Reflect.construct`.

Definitely not. But it's not related this PR.

Reply:
Which browsers support Proxy but not Reflect.construct?

Reply:
For example, FF18?

Reply:
Oh ok, it's the other way. If a browser supports Reflect.construct, it also supports Proxy.